open-metadata-resources/open-metadata-deployment/charts/vdc/templates/job-01-data-loader.yaml

```diff
@@ -0,0 +1,83 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
The official PG image has the option to run db scripts. there was a bug I reported, now supposedly fixed. BUT I think we have better flexibility doing it this way (and the job is a good change)